pinc/ProjectTransition.inc

```diff
@@ -681,26 +681,6 @@ for ($rn = 1; $rn <= MAX_NUM_PAGE_EDITING_ROUNDS; $rn++) {
         ];
 
         foreach ($from_states as $from_state) {
-            if ($rn > 1) {
-                // Transition to go back 1 round.
-
-                $prev_round = get_Round_for_round_number($rn - 1);
-
-                new ProjectTransition(
```

Comment:
removing this transition doesn't impact the script you mentioned in the description, right?

Reply:
Good question. I don't think it should, but it's a good idea for me to verify that, thanks. I'll investigate that.

Reply:
Double-checked by actually trying it out. No, it doesn't impact the script. The change doesn't actually remove the plumbing that would allow the transition to happen, it just removes the button that would cause the transition to be performed. The `noncvs` script that actually does the round repeat does not need the button to be defined.